README.md

```diff
@@ -115,6 +115,14 @@ helm template eirini-controller "deployment/helm" \
   --set "images.eirini_controller=eirini/eirini-controller@sha256:42e22b3222e9b3788782f5c141d260a5e163da4f4032e2926752ef2e5bae0685" \
   --namespace "eirini-controller" | kubectl apply -f -
 ```
+---
+### Install Hierarchical Namespaces Controller
```

Comment:
I suspect that we also need to tell users to configure HNC to propagate secrets so that the kpack image registry secret is available in the space namespaces. Here is the [corresponding command](https://github.com/cloudfoundry/cf-k8s-controllers/blob/9a978623c7153f7b7575f2269438926b1b3174fe/scripts/install-dependencies.sh#L136) from the `install-dependencies.sh` script.

This will mean that we also need to include instructions to install the `kubectl-hns` plugin. Here is the [corresponding command](https://github.com/cloudfoundry/cf-k8s-controllers/blob/9a978623c7153f7b7575f2269438926b1b3174fe/scripts/install-dependencies.sh#L108) from the `install-dependencies.sh` script.

Reply:
@akrishna90 and I **think** we have addressed this. PTAL.